### ISSUE TITLE
Weight decay was not applied

### ISSUE THREAD

**Issue by dawenl** [williamleif/GraphSAGE#21]
In this line:

https://github.com/williamleif/GraphSAGE/blob/75794f8a09ec25d98bc00a899c58df63e033b9e7/graphsage/models.py#L390

The loss is re-assigned to the link prediction loss. It should be `self.loss += ...`

(It doesn't affect anything since the weight decay is set to 0 anyways)

**Comment by williamleif** [comment]
Thanks for the catch! I just pushed a fix, though as a note, we never found that weight decay helped :)
